src/spec_kernels.cpp

```diff
@@ -0,0 +1,339 @@
+#include "../include/spec_kernels.hpp"
+
+// Declare loops
+#define PARALLEL_FOR_2D                                   \
```

Comment:
Please add here a header with a list of gates which currently can use the specialized implementation 2

Reply:
Currently, it is in the documentation of `TurnOnSpecializeV2` and available in the Doxygen. You can see `qureg_init.cpp` for the documentation.